src/index.ts

```diff
@@ -24,7 +36,7 @@ class CumulusMessageAdapterExecutionError extends Error {
  * @param {string} command - the action to be performed by the message-adapter
  * @returns {Promise.<Array>} - Returns arguments used to spawn the CMA
  */
-async function generateCMASpawnArguments(command) {
+async function generateCMASpawnArguments(command: string): Promise<[string, string[]]> {
   const adapterDir = process.env.CUMULUS_MESSAGE_ADAPTER_DIR || './cumulus-message-adapter';
```

Comment:
Make sure we don't get an empty string.

```suggestion
  const adapterDir = process.env.CUMULUS_MESSAGE_ADAPTER_DIR ?? './cumulus-message-adapter';
```